src/ts/components/listbox.ts

```diff
@@ -0,0 +1,119 @@
+import {Button, ButtonConfig} from './button';
+import {ListSelector, ListSelectorConfig} from './listselector';
+import {DOM} from '../dom';
+
+/**
+ * A element to select a single item out of a list of available items.
+ *
+ * DOM example:
+ * <code>
+ *   <div class='ui-listbox>
+ *     <div class='ui-listboxitem-wrapper>
+ *       <div class='ui-listboxitem'>
+ *         <button class='ui-listboxitem-button'>label</button>
+ *       </div>
+ *     </div>
+ *     ...
+ *   </div
+ * </code>
+ */
+export class ListBox extends ListSelector<ListSelectorConfig> {
+  private listBoxElement: DOM;
```

Comment:
This field is unnecessary. The `DOM` object is already stored in `Component` and can be retrieved with `getDomElement()`